src/earn/EarnEnterAmount.tsx

```diff
@@ -619,6 +632,33 @@ function TransactionWithdrawDetails({
           </Text>
         </View>
       </View>
+      {pool.dataProps.withdrawalIncludesClaim &&
+        rewardsPositions.map((position, index) => (
+          <View key={index} style={styles.txDetailsLineItem}>
```

Comment:
might be a bit tricky since some show crypto amount in parentheses but some don't, some have multiple line items to be shown but some don't. I can make a follow-up ticket for cleanup if that would be helpful.

Reply:
https://linear.app/valora/issue/ACT-1431/placeholder-clean-up-earnenteramount-line-items